Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -69,9 +69,12 @@ void VideoConfig::Load(const std::string& ini_file)
   settings->Get("OverlayStats", &bOverlayStats, false);
   settings->Get("OverlayProjStats", &bOverlayProjStats, false);
   settings->Get("DumpTextures", &bDumpTextures, false);
-  settings->Get("HiresTextures", &bHiresTextures, false);
-  settings->Get("ConvertHiresTextures", &bConvertHiresTextures, false);
-  settings->Get("CacheHiresTextures", &bCacheHiresTextures, false);
+  if (!settings->Get("CustomTextures", &bCustomTextures, false))
+  {
+	  settings->Get("HiresTextures", &bCustomTextures, false);
+  }
+  settings->Get("ConvertCustomTextures", &bConvertCustomTextures, false);
+  settings->Get("CacheCustomTextures", &bCacheCustomTextures, false);
```

Comment:
If you're going to have migration code for "CustomTextures", you are also supposed to have migration code for "ConvertCustomTextures" and "CacheCustomTextures".